test/integration/share/inspec.yml

```
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+name: shared
+title: InSpec shared resources
```

Similar to [this situation](https://github.com/saltstack-formulas/template-formula/pull/203/files#r481273581), this differs from all of the other `test/integration/share/inspec.yml` files out there.  The current template used by the `ssf-formula` produces:

```suggestion
name: share
title: TEMPLATE formula
```

The `name` must be changed (to match the actual suite name itself) but we need to decide what to do about the `title`, since it will affect all of the other formulas as well.